### ISSUE TITLE
Train road AI for waterways too

### ISSUE THREAD

**Issue by Artoria2e5** [facebookincubator/RapiD#198]
From some HOT mapping it is kinda common for me to see the AI thing confusing waterways for track roads. It is bad data, but I do appreciate something having done the job of tracing lines for me.

Maybe the AI should just be trained with a category of waterway=stream? Streams look like track roads, but they usually are curvier, and tend to come with a bunch of tell-tale plant cover to the sides. I think the AI can tell them apart given correctly labeled data. (For human editors in RapiD, the OpenTopomap SRTM data can be checked foe a valley too.)

From a utilitarian perspective waterways are quite important for disaster management too. They also help provide a key metric for evaluating roads.

**Comment by nitramwin** [comment]
I fully agree. I cannot imagine that this hasn't been discussed by the team in the past.

**Comment by jjiglesiasg** [comment]
Will be beautiful such feature. However right now several water courses are recognized by Rapid as roads, it{s up to the mapper to convert them to waterways...

**Comment by jjiglesiasg** [comment]
PLEASEEEEE!!!

**Comment by JLZIMMERMANN** [comment]
this is really waited since this wame year
